language/cypher/src/main/java/com/neueda/jetbrains/plugin/graphdb/language/cypher/util/FileTypeExtensionUtil.java

```diff
@@ -0,0 +1,18 @@
+package com.neueda.jetbrains.plugin.graphdb.language.cypher.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileTypeExtensionUtil {
+
+    public static final List<String> EXTENSIONS = new ArrayList<String>() {{
```

Comment:
Use `Arrays.asList`

Reply:
fixed in [d787900e](https://github.com/ex3m1024/jetbrains-plugin-graph-database-support/commit/d787900e73bc18d48e9b0ac186e839f178cb809b)